shared/messaging/utils.go

```diff
@@ -0,0 +1,26 @@
+package messaging
```

Comment:
Though I'm also at fault of always using `utils.go`, I think `envelope.go` could help scope from becoming a catchall.

Weakly held point so lmk what you think.